CollAction/package.json

```diff
@@ -3,33 +3,40 @@
   "version": "1.0.0",
   "description": "CollAction Front End Application",
   "main": "Frontend/app/index.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/CollActionteam/CollAction"
+  },
   "scripts": {
     "build": "webpack",
     "watch": "webpack -w"
   },
-  "author": "Stewart Matheson",
+  "author": "CollAction",
   "license": "ISC",
   "devDependencies": {
-    "@types/react": "^15.0.6",
-    "@types/react-dom": "^0.14.22",
-    "@types/react-tap-event-plugin": "0.0.30",
-    "css-loader": "^0.26.1",
-    "jquery": "^3.1.1",
-    "node-sass": "^4.9.0",
-    "react": "^15.4.2",
-    "react-dom": "^15.4.2",
-    "react-tap-event-plugin": "^2.0.1",
-    "sass-loader": "^4.1.1",
-    "scss": "^0.2.4",
-    "style-loader": "^0.13.1",
-    "ts-loader": "^1.3.3",
-    "tslint": "^4.4.2",
-    "typescript": "^2.1.5",
-    "webpack": "^1.14.0"
+    "@types/jquery": "^3.3.5",
+    "@types/jquery-validation-unobtrusive": "^3.2.32",
```

Comment:
If we're loading unobtrusive validation through npm, do we still the cdn links?

Reply:
Moved the validation code into the typescript bundle, so loading from CDN is no longer needed. If also solves the need for "if $ != undefined" statements in the validation scripts.